docs/tutorial/complete-bicep-files/06.bicep

```diff
@@ -15,9 +15,9 @@ var contributor = 'b24988ac-6180-42a0-ab88-20f7382dd24c'
 resource rbac 'Microsoft.Authorization/roleAssignments@2020-04-01-preview' = {
   name: guid(subscription().id, objectId, contributor)
   properties: {
-    roleDefinitionId: contributor
+    roleDefinitionId: subscriptionResourceId('microsoft.authorization/roleAssignments', contributor)

```

Comment:
Thoughts on using the `existing` syntax here instead? I'm generally keen to avoid pointing people to using the `*resourceId` functions as they're somewhat error prone.

Reply:
good call - fixed.